portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,5 +50,42 @@ private String convertToJson(ArrayList<String> values) {
     json += "\"" + values.get(2) + "\"";
     json += "}";
     return json;
+  }*/
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "Comment-Input", "");
+    boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
+    boolean lowerCase = Boolean.parseBoolean(getParameter(request, "lower-case", "false"));
+
+    // Convert the text to upper case.
+    if (upperCase) {
+      text = text.toUpperCase();
+    }
+
+    // Convert the text to lower case.
+    if (lowerCase) {
+      text = text.toLowerCase();
+    }
+
+    // Break the text into individual words.
+    String[] words = text.split("\\s*,\\s*");
```

Comment:
why does the text need to be split into individual words?

Reply:
Yes. It does not need to.